db/data_migration/20141125162430_ensure_all_documents_have_content_id.rb

```diff
@@ -1 +1,3 @@
-ActiveRecord::Base.connection.execute 'UPDATE documents SET content_id=UUID() where content_id IS NULL'
+Document.where(content_id: nil).each do |document|
```

Comment:
Worth doing a `find_each` here to avoid loading them all into memory simultaneously?  (http://apidock.com/rails/ActiveRecord/Batches/find_each)